README.md

```diff
@@ -1,29 +1,234 @@
 # profiles
 Gitops native package management
 
-## Installation on a local cluster using [Kind](https://kind.sigs.k8s.io/)
+<!--
+To update the TOC, install https://github.com/kubernetes-sigs/mdtoc
+and run: mdtoc -inplace README.md
+-->
+
+ <!-- toc -->
+- [Getting started](#getting-started)
+  - [Local environment using <a href="https://kind.sigs.k8s.io/">Kind</a>](#local-environment-using-kind)
+  - [Local development](#local-development)
+  - [Installing Profiles](#installing-profiles)
+- [Terminology](#terminology)
+  - [Profile](#profile)
+  - [Catalog](#catalog)
+  - [Profile Catalog Source](#profile-catalog-source)
+  - [Profile Subscription](#profile-subscription)
+  - [Profile Controller](#profile-controller)
+  - [Profile Catalog Source Controller](#profile-catalog-source-controller)
+- [Current Architecture](#current-architecture)
+  - [Catalogs and Sources](#catalogs-and-sources)
+  - [Profile Subscriptions](#profile-subscriptions)
+- [Roadmap](#roadmap)
+  - [Profiles](#profiles)
+  - [Catalogs](#catalogs)
+<!-- /toc -->
+
+## Getting started
+
+### Local environment using [Kind](https://kind.sigs.k8s.io/)
+
 1. Set up local environment: `make local-env`.
 
     This will start a local `kind` cluster and installs
     the `profilesubscription`, `source` and `helm` controllers.
 
-1. deploy an example catalog `kubectl apply -f examples/profile-catalog-source.yaml`
+1. Deploy an example catalog source `kubectl apply -f examples/profile-catalog-source.yaml`
+
+1. Expose the catalog API: `kubectl -n profiles-system port-forward $(kubectl get pods -A | awk '/profiles-controller-manager/ {print $2}') 8000:8000`
+
+### Local development
+
+1. Tests can be run with `make test`. (note: the previous env setup needs to be done first.)
```

Comment:
```suggestion
1. Tests can be run with `make test`.
```

env only needed for acceptance I think?

Reply:
oh yeh, and the acceptance command will do local env anyway